### DISCUSSION TITLE
Windows on ARM (or 32bits) version

### DISCUSSION THREAD

**Discussion by brennomeneses** [Kong/insomnia#5150] [Ideas]
Hello everyone! 
I recently bought a Galaxy Book Go, with Snapdragon 7C Gen 2, but when I setup my dev envoriment, I got a error while installing insomnia, I think this is a ARM error, because installing other Electron apps like Caret.io and Discord, I got no problems.
Checkout the error:
[Image of error](https://imgur.com/a/YwsmZ5Z)
[Error log (Ghostbin)](https://ghostbin.me/6319273711765)

**Comment by jackkav** [comment]
Duplicate of #3741
